scanner/src/main/kotlin/Main.kt

```diff
@@ -210,41 +212,43 @@ object Main {
             "Provided path is not a file: ${dependenciesFile.absolutePath}"
         }
 
-        val includedScopes = sortedSetOf<Scope>()
-        val excludedScopes = sortedSetOf<Scope>()
-        val analyzerErrors = sortedMapOf<Identifier, List<String>>()
-
         val mapper = dependenciesFile.mapper()
 
-        val projectAnalyzerResult = mapper.readValue(dependenciesFile, ProjectAnalyzerResult::class.java)
-        analyzerErrors.putAll(projectAnalyzerResult.collectErrors())
+        val analyzerResult = try {
```

Comment:
I think this deserves a code comment: We try the legacy `ProjectAnalyzerResult` first, then fall back to the new `AnalyzerResult`. From a logical perspective, shouldn't we switch to make the fall-back path contain the code to read the legacy result format?

Reply:
Done